architecture/core_concepts/routes.adoc

```diff
@@ -12,22 +12,31 @@ toc::[]
 
 == Overview
 
-An OpenShift route is a way to expose a
-link:pods_and_services.html#services[service] by giving it an
-externally-reachable hostname like `www.example.com`.
+An OpenShift link:../../architecture/core_concepts/routes.html[route] exposes a
```

Comment:
Thanks for letting me know. I went through and made the changes and verified the links work.